BD_funcoes.py

```diff
@@ -1,17 +1,17 @@
 from mysql.connector import connect
 
-dados_database = {"host":"localhost", "user":"root", "password":"root", "database":"bluecommerce",}
+dados_database = {"host":"localhost", "user":"root", "password":"root", "database":"bluecommerce"}
 
 
 def query(sql, params=None):
-    with connect(**dados_database) as conn:
-        with conn.cursor() as cursor:
+    with connect(host = "localhost", user = "root", password = "root", database = "bluecommerce") as conn:
+        with conn.cursor(dictionary=True) as cursor:
             cursor.execute(sql, params)
             return cursor.fetchall()
 
 
 def execute(sql, params=None):
-    with connect(**dados_database) as conn:
+    with connect(host="localhost", user="root", password="root", database="bluecommerce") as conn:
```

Comment:
Tinha dado algum erro aqui que fez você mudar?
```suggestion
    with connect(**dados_database) as conn:
```